e2e/playwright/.eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
+    "extends": [
+        "../../webapp/.eslintrc.json"
```

Comment:
I tried to extend from mattermost-server/webapp/.eslintrc.json but has no rules at all, and missed most of the points I wanted to cover. That's why I extended from plugin/webapp/.eslintrc.json